lib/src/native/realm_core.dart

```diff
@@ -164,12 +315,48 @@ class ConfigHandle extends Handle<realm_config> {
 
 class RealmHandle extends Handle<shared_realm> {
   RealmHandle._(Pointer<shared_realm> pointer) : super(pointer) {
-    _realmLib.realm_attach_finalizer(this, _pointer.cast(), 1);
+    _realmLib.realm_attach_finalizer(this, _pointer.cast(), 24);
   }
 }
 
 class SchedulerHandle extends Handle<realm_scheduler> {
   SchedulerHandle._(Pointer<realm_scheduler> pointer) : super(pointer) {
-    _realmLib.realm_attach_finalizer(this, _pointer.cast(), 1);
+    _realmLib.realm_attach_finalizer(this, _pointer.cast(), 24);
+  }
+}
+
+class RealmObjectHandle extends Handle<realm_object> {
+  RealmObjectHandle._(Pointer<realm_object> pointer) : super(pointer) {
+    _realmLib.realm_attach_finalizer(this, _pointer.cast(), 112);
+  }
+}
+
+extension _StringEx on String {
```

Comment:
You can drop `_StringEx` and just do `extension on String` since it is private anyway

Reply:
👍